public/app/features/plugins/angularDeprecation/AngularDeprecationNotice.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { Alert } from '@grafana/ui';
+import { LocalStorageValueProvider } from 'app/core/components/LocalStorageValueProvider';
+
+const LOCAL_STORAGE_KEY_PREFIX = 'grafana.angularDeprecation.dashboardNotice.isDismissed';
+
+function localStorageKey(dashboardUid: string): string {
+  return LOCAL_STORAGE_KEY_PREFIX + '.' + dashboardUid;
+}
+
+export interface Props {
+  dashboardUid: string;
+}
+
+export function AngularDeprecationNotice({ dashboardUid }: Props) {
+  return (
+    <LocalStorageValueProvider<boolean> storageKey={localStorageKey(dashboardUid)} defaultValue={false}>
```

Comment:
having the storage key include uid means it could show up on all dashboards potentially and have to be dismissed A LOT by every user.